canonical-web-and-design/maas-ui
# Implement adding and editing repositories

## Done
- Implemented adding and editing repositories

## QA
- Click "Add repository", fill in the forms correctly and click "Save repository". Check that you are redirected to the repo list and the new repo is there with a notification.
- Click the edit button for the repo and edit some fields. Click "Save repository" and check that you are redirected to the repo list with a notification.
- Click "Add repository" and use the same name you used before. Click "Save repository" and check that a server error shows underneath the Name field.
- Delete a repo and see there is a notification.
- In another browser tab, make some changes to repos from the old maas settings and check that the details are updated in real-time.
- Do the same with users because I've used a couple of custom hooks there too
